Azure/sonic-buildimage
# fix Z9100 port index issue

**- What I did**
Fixed port indexing issue in Z9100 for AP

**- How I did it**
Used overriding method to fix the port index issue.
**- How to verify it**
Check sfpshow presence,eeprom and do OIR,verify whether the appropriate port status is getting updated or not.
**- Description for the changelog**
UT log:[z9100_indexing_UT_logs.txt](https://github.com/Azure/sonic-buildimage/files/4367472/z9100_indexing_UT_logs.txt)
